test/test_torch.py

```diff
@@ -5634,6 +5634,64 @@ def test_index_add(self):
                             dest2[idx[i]] += src[i] * 2
                         self.assertEqual(dest, dest2)
 
+    # FIXME: move to indexing test suite
+    def test_index_reductions(self):
+        device = 'cpu'
```

Comment:
nit: if this becomes a parameter to the function via parametrize the test in the PR up the stack up will become simpler